packages/components/src/components/modal/_modal.scss

```diff
@@ -49,6 +49,7 @@
         visibility 0ms linear;
     }
 
+    // V11: Question for design: do we have an updated tokens for fields that exist on `layer`?
```

Comment:
@aagonzales does this have an answer or this an open question?

Reply:
I talked to @tw15egan about this. Fields that exist on a layer would be styled using an inline theme now. So it would still be `$field` just from the theme in the next step in the layering modal. @joshblack talked about potentially building some of this inline theming logic into the components. 

Some products teams have argued for keeping a `$field-02` and light variant in each theme so each theme is a little more independent. This is a big question we're hoping to answer with our alpha and beta testing for the major release.

Reply:
For now, I've left a little `// V11: possibly deprecate` note near all usage of `light` specific tokens so we can easily find them once we determine a path 👍🏻